content/en/docs/Installation/trustedboot.md

```diff
@@ -0,0 +1,198 @@
+---
+title: "Trusted Boot Installations"
+linkTitle: "Trusted Boot"
+weight: 6
+date: 2022-11-13
+description: >
+---
+
+"Trusted Boot" is a combination of technologies that allows us to guarantee that a system was not tampered with, and the user-data is protected by cold attacks, it is composed by FDE, Secure Boot and Measured Boot.
+
+If you want to learn more on what Trusted Boot is and how it works, see the [Trusted Boot Architecture]({{< relref "../architecture/trustedboot" >}}) page. This page describes how to enable Trusted Boot support in Kairos.
+
+## Enable Trusted Boot in Kairos
+
+Kairos supports Trusted boot by generating special installable medium, and we boot into what we call “UKI-mode”. [UKI](https://uapi-group.org/specifications/specs/unified_kernel_image/) is a specific file format tailored to achieve a tamper-proof system and encryption of user-data bound to the silicon by relying on HW capabilities. This feature is optional and works alongside how Kairos works.
+
+> UKI stands for “Unified Kernel Images”. UKI files are a single, fat binary that encompasses the OS and the needed bits in order to boot the full system with a single, verified file. You can read technical details here: Brave New Trusted Boot World.
```

Comment:
Maybe we should mention here that we use USI?

Reply:
done!